src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -95,9 +95,7 @@
 logger = init_logger(__name__)
 service_metrics = ServiceMetrics()
 
-ADD_SPECIAL_TOKENS = os.getenv("ADD_SPECIAL_TOKENS")
-if ADD_SPECIAL_TOKENS is not None:
-    ADD_SPECIAL_TOKENS = ADD_SPECIAL_TOKENS.lower() not in (0, "false")
+ADD_SPECIAL_TOKENS = os.getenv("ADD_SPECIAL_TOKENS", "true").lower() not in (0, "false")
```

Comment:
Since `os.getenv` always returns a string, I think this should be 
```suggestion
ADD_SPECIAL_TOKENS = os.getenv("ADD_SPECIAL_TOKENS", "true").lower() not in ("0", "false")
```

Reply:
Good point!

Reply:
Thanks, yes this was my mistake originally!